pkg/ws/report/handlers/initial_request_test.go

```diff
@@ -0,0 +1,96 @@
+package wshandlers
+
+import (
+	"encoding/json"
+	"github.com/kptm-tools/core-service/pkg/domain"
+	"github.com/kptm-tools/core-service/pkg/dto"
+	"github.com/kptm-tools/core-service/pkg/interfaces"
+	"github.com/kptm-tools/core-service/pkg/mocks"
+	"github.com/kptm-tools/core-service/pkg/ws/common"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestInitialRequestHandler(t *testing.T) {
+	dataBadScanId, _ := json.Marshal(&dto.InitialDataRequest{ScanID: ""})
+	dataGoodScanId, _ := json.Marshal(&dto.InitialDataRequest{ScanID: "56a9b230-1a67-40c1-ad97-603ebf304841"})
```

Comment:
Audit: 

* pkg/ws/report/handlers/initial_request_test.go:15:2: var dataBadScanId should be dataBadScanID (ST1003)
* pkg/ws/report/handlers/initial_request_test.go:16:2: var dataGoodScanId should be dataGoodScanID (ST1003)

```suggestion
	dataBadScanID, _ := json.Marshal(&dto.InitialDataRequest{ScanID: ""})
	dataGoodScanID, _ := json.Marshal(&dto.InitialDataRequest{ScanID: "56a9b230-1a67-40c1-ad97-603ebf304841"})
```